### PR TITLE
parallelio: New Package.

### PR DESCRIPTION
**Pull request by tkameyama**

### CODE REVIEW

File: var/spack/repos/builtin/packages/parallelio/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Parallelio(CMakePackage):
+    """he Parallel IO libraries (PIO) are high-level parallel I/O C and
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/parallelio/package.py]:
```suggestion
    """The Parallel IO libraries (PIO) are high-level parallel I/O C and
```

File: var/spack/repos/builtin/packages/parallelio/package.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Fortran libraries for applications that need to do netCDF I/O from
+    large numbers of processors on a HPC system."""
+
+    homepage = "https://ncar.github.io/ParallelIO/"
+    url      = "https://github.com/NCAR/ParallelIO/archive/pio2_5_2.tar.gz"
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/parallelio/package.py:16]:
```suggestion

    maintainers = ['tkameyama']
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
LGTM.  Just a couple of suggestions.

File: var/spack/repos/builtin/packages/parallelio/package.py (hunk 1 above)
Comment (tkameyama) [reply var/spack/repos/builtin/packages/parallelio/package.py]: fixed.

File: var/spack/repos/builtin/packages/parallelio/package.py (hunk 2 above)
Comment (tkameyama) [reply var/spack/repos/builtin/packages/parallelio/package.py:16]: done.

**Review by tldahlgren — APPROVED** [review]
LGTM

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**